### PR TITLE
Add wstETH as collateral to USDC market on Mainnet

### PR DESCRIPTION
**Pull request by MishaShWoof**

### CODE REVIEW

File: scenario/MainnetBulkerScenario.ts [hunk 1]
```diff
@@ -10,21 +10,43 @@ import {
...
+
+async function getWstETHIndex(context: any): Promise<number> {
+  const comet = await context.getComet();
+  const totalAssets = await comet.numAssets();
+  for (let i = 0; i < totalAssets; i++) {
+    const asset = await comet.getAssetInfo(i);
+    if (asset.asset.toLowerCase() === MAINNET_WSTETH_ADDRESS) {
+      return i;
+    }
+  }
```
Comment (dmitriy-woof-software) [inline scenario/MainnetBulkerScenario.ts:24]: If no wstETH, let's return `-1`

File: scenario/MainnetBulkerScenario.ts [hunk 2]
```diff
@@ -10,21 +10,43 @@ import {
...
+}
+
+async function hasWstETH(context: any): Promise<boolean> {
+  return(await getWstETHIndex(context) !== undefined);
```
Comment (dmitriy-woof-software) [inline scenario/MainnetBulkerScenario.ts:28]: check, if `getWstETHIndex(context) > -1`

File: scenario/MainnetBulkerScenario.ts [hunk 3]
```diff
@@ -10,21 +10,43 @@ import {
...
+      {
+        albert: { [`$asset${await getWstETHIndex(ctx)}`]: '== 0' },
+      }
+    ),
   },
   async ({ comet, actors, bulker }, context) => {
     const { albert } = actors;
 
-    const stETH = await context.world.deploymentManager.contract('stETH') as ERC20;
+    const stETH = await context.world.deploymentManager.hre.ethers.getContractAt('ERC20', MAINNET_STETH_ADDRESS) as ERC20;
```
Comment (dmitriy-woof-software) [inline scenario/MainnetBulkerScenario.ts:49]: `deploymentManager.existing`

File: deployments/mainnet/usdc/migrations/1720623615_add_wsteth_as_collateral.ts [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { exp, proposal } from '../../../../src/deploy';
+
+const WSTETH_ADDRESS = '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0';
+const STETH_USD_PRICE_FEED_ADDRESS = '0xCfE54B5cD566aB89272946F602D76Ea879CAb4a8';
+let newPriceFeedAddress: string;
+
+export default migration('1720623615_add_wsteth_as_collateral', {
+  async prepare(deploymentManager: DeploymentManager) {
+    const _wstETHScalingPriceFeed = await deploymentManager.deploy(
```
Comment (dmitriy-woof-software) [inline deployments/mainnet/usdc/migrations/1720623615_add_wsteth_as_collateral.ts:12]: Use existing price feed. Have a look on USDT Mainnet deployment.

**Comment by dmitriy-woof-software** [comment]
Proposal - https://compound.finance/governance/proposals/286

**Review by torreyatcitty — APPROVED** [review]
LGTM

**Review by torreyatcitty — APPROVED** [review]

**Pull request merged**